app/src/main/java/com/geeksville/mesh/model/NodeDB.kt

```diff
@@ -60,6 +66,22 @@ class NodeDB @Inject constructor(
 
     fun myNodeInfoFlow(): Flow<MyNodeInfo?> = nodeInfoDao.getMyNodeInfo()
     fun nodeInfoFlow(): Flow<List<NodeInfo>> = nodeInfoDao.getNodes()
+    fun getNode(num: String): NodeInfo {
+        val node: NodeInfo? = nodes.value[num];
+        if (node != null) {
+            return node;
+        } else {
+            return NodeInfo(
+                DataPacket.idToDefaultNodeNum(num),
+                MeshUser(
+                    id = num,
+                    longName = app.getString(R.string.unknown_username, num),
+                    shortName = app.getString(R.string.unknown_node_short_name),
+                    hwModel = HardwareModel.UNSET,
+                )
+            );
+        }
+    }
```

Comment:
this should be avoided. overriding database results can lead to consistency issues.

also in Kotlin semicolons are not used.